src/sbt-test/sbt-doctest/html-entities/build.sbt

```diff
@@ -1,10 +1,9 @@
-crossScalaVersions := Seq("2.10.7", "2.11.12", "2.12.6")
+crossScalaVersions := Seq("2.11.12", "2.12.6")
```

Comment:
Why did you removed 2.10.7 here? I suppose 2.10 can be supported as scalaVersion and crossScalaVersion because scalameta works as sbt plugin.

Reply:
specs2 4.x dropped 2.10 support. I can revert specs to 3.x version, but I think 2.10 has to go anyway, we gonna get 2.13 soon. 
or, one way can be do specs 3.x for 2.10 version and 4.x for 2.11+, but i'm not sure it's worth it, ecosystem need to move forward i think
lmk what you think

Reply:
I'd like to support 2.10 if it become too hard/complex. One of our user typelevel/cats still uses sbt-doctest with scala 2.10.

Reply:
actually cats just dropped 2.10 in 1.2.0 release (https://github.com/typelevel/cats/blob/master/CHANGES.md)

Reply:
and it works for scalatest anyway, but i will see what i can do for 2.10 and specs

Reply:
that was easy enough! please review again